src/rocker/cli.py

```diff
@@ -50,6 +55,12 @@ def main():
     args = parser.parse_args()
     args_dict = vars(args)
 
+    # Load config file if provided
+    if args.config:
+        with open(args.config, 'r') as f:
+            config = yaml.safe_load(f)
```

Comment:
I feel like we should be doing some validation here. 

And providing some visibility as to what's being loaded and from where as well. Think about how one would debug a typo in a config value or confirm that something is being selected.